incapsula/resource_subaccount.go

```diff
@@ -52,6 +52,14 @@ func resourceSubAccount() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringInSlice([]string{"full", "security", "none", "default"}, false),
 			},
+			"data_storage_region": {
+				Description:  "Default data region of the sub account for newly created sites. Options are `APAC`, `EU`, `US` and `DEFAULT`. Defaults to `DEFAULT`.",
+				Type:         schema.TypeString,
+				Default:      "US",
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"APAC", "EU", "US", "AU"}, false),
+				ForceNew:     true,
```

Comment:
why force new?

Reply:
Since we cannot update sub account we must set all fields as ForceNew

Reply:
Done